readme.md

```diff
@@ -150,3 +150,48 @@ The best time for the whole team to start is at the "ideation" or "Design Sprint
 stage so that no time is wasted on this on-boarding later on.
 
 Further reading: https://en.wikipedia.org/wiki/Brooks%27s_law
+
+## When to create a new issue when dealing with bugs 🐛  and enhancements :ribbon:
+
+An existing issue is put into `please-test` as the team have finished working on
+it and all of the acceptance criteria have been fulfilled. You, the product
+owner test the issue and on reflection you decide that you think the designs
+would look better with a couple of tweaks - maybe removing the bold on the
+title, making the logo a bit bigger and changing the copy by a few words. What
+do you do in this situation?
+
+These changes might relate to the changes made in the issue but if they are
+fundamentally not what you asked for in the first place or not what was shown on
+the mock-up then they belong in a new issue **NOT** as a 'bug' or as a request
+at the bottom of the existing issue. Small as they seem, these issues are
+enhancements and new scope and they should be dealt with separately.
+
+### Why?
+These kinds of small changes add up, especially if you change your mind a couple
+of times or if there are a few across multiple issues. Not only do the github
+issues become muddied as their title may no longer reflect the content inside
+but also it becomes more complicated for people to follow the chains of
+conversations that have evolved. Also, the extra changes will impact the
```

Comment:
Got a little cross-eyed reading this big block of text - it could do with some additional highlighting with some boldening of important words and phrases . We know POs are unlikely to read everything here!
https://github.com/dwyl/start-here/issues/118#issuecomment-278950926